tests/unit/dojson/test_dojson_conferences.py

```diff
@@ -580,26 +582,35 @@ def test_series_number_from_411__n():
         '</datafield>'
     )  # record/1447029
 
-    expected = 7
     result = clean_record(conferences.do(create_record(snippet)))
 
-    assert expected == result['series_number']
+    expected = [
+        {
+            'number': 7
+        },
+    ]
+    assert result['series'] == expected
```

Comment:
This is asserting that you're creating invalid records, according to the schema above. Definitely not a good thing, since now we have 3.4k more invalid records: https://sentry.cern.ch/inspire-sentry/inspire-labs-qa/group/603955/.